code/__DEFINES/maps.dm

```diff
@@ -1,23 +1,36 @@
 /*
-The /tg/ codebase allows mixing of hardcoded and dynamically-loaded z-levels.
+The /tg/ codebase allows mixing of hardcoded and dynamically-loaded Z-levels.
 Z-levels can be reordered as desired and their properties are set by "traits".
-See map_config.dm for how a particular station's traits may be chosen.
+See code/datums/map_config.dm for how a particular station's traits may be chosen.
 The list DEFAULT_MAP_TRAITS at the bottom of this file should correspond to
 the maps that are hardcoded, as set in _maps/_basemap.dm. SSmapping is
-responsible for loading every non-hardcoded z-level.
+responsible for loading every non-hardcoded Z-level.
 
-As of 2018-02-04, the typical z-levels for a single-level station are:
+As of April 26th, 2022, the typical Z-levels for a single-level station are:
 1: CentCom
 2: Station
-3-4: Randomized space
+3-4: Randomized Space (Ruins)
 5: Mining
-6: City of Cogs
-7-11: Randomized space
-12: Empty space
-13: Transit space
-
-Multi-Z stations are supported and multi-Z mining and away missions would
-require only minor tweaks.
+6-11: Randomized Space (Ruins)
+12: Transit Space
+
+However, if away missions are enabled:
+12: Away Mission
+13: Transit/Reserved Space
+
```

Comment:
You do not include reserved space in your initial list

Reply:
That's what it says in-game, is there some distinction that I'm missing?

Reply:
Reserved space is by nature transit space, The two are one and the same. It holds basically anything that requests space to use.
![image](https://user-images.githubusercontent.com/58055496/166616153-3cb25119-6a5a-45b7-90dc-9fa6d213f28d.png)
It can scale up and down in size, but it should always be called Transit/Reserved, since it's created as a part of /datum/controller/subsystem/mapping/Initialize.